src/ostree/ot-admin-builtin-status.c

```diff
@@ -31,7 +31,10 @@
 
 #include <glib/gi18n.h>
 
+static gboolean opt_verify;
+
 static GOptionEntry options[] = {
+  { "verify", 'V', 0, G_OPTION_ARG_NONE, &opt_verify, "Print the commit verification status", NULL },
```

Comment:
I'm not familiar with signed setups, but this does not seem to offer a way to choose between GPG and signapi verification. Are they usually either both enabled or both disabled? Wouldn this be better suited as a verb with flags for different methods?

Reply:
I think using both is supported, but not expected (except perhaps as a transition period).  But if both are enabled in the remote config, then I'd generally expect us to do both.

I can't think of a need for a CLI consumer to explicitly just check one.